### PR TITLE
CTC  target un-pad example

### PR DESCRIPTION
**Pull request by MLgdg**
CTC  target un-pad example

### CODE REVIEW

**Comment by ezyang** [comment]
@t-vi do you think you could review this?

**Comment by MLgdg** [comment]
pr/py3.6-clang7-rocmdeb-ubuntu16.04 — Build failed

**Comment by MLgdg** [comment]
Failed: pr/caffe2-py3.6-devtoolset7-rocmrpm-centos7-test

**Comment by t-vi** [comment]
@MLgdg: Don't worry about the CI.

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -1266,12 +1266,13 @@ class CTCLoss(_Loss):
         - Output: scalar. If :attr:`reduction` is ``'none'``, then
           :math:`(N)`, where :math:`N = \text{batch size}`.
 
-    Example::
+    Example1::
```
Comment (t-vi) [inline torch/nn/modules/loss.py:1269]: This isn't just a free-form headline, but a tag from a fixed vocabulary. You want to have `Examples::` here and then use comments to split the examples
Please search for `Examples::` in this file for how it is done for other losses.

File: torch/nn/modules/loss.py [hunk 2]
```diff
@@ -1286,6 +1287,24 @@ class CTCLoss(_Loss):
         >>> loss = ctc_loss(input, target, input_lengths, target_lengths)
         >>> loss.backward()
 
+    Example2::
```
Comment (t-vi) [inline torch/nn/modules/loss.py:1290]: See above.

**Review by t-vi — CHANGES_REQUESTED** [review]
Hello @Mlgdg.

First off, thank you for improving PyTorch's.
Using CTC loss has been a source of confusion for many users and it is great that you help them along.

There is a small change I would need you to do: I think the Example1 and Example2 headings don't quite work that way. Could you change it to a single Examples heading?

Best regards and thank you again

Thomas

**Review by t-vi — APPROVED** [review]
Looks good! Thank you very much.

File: torch/nn/modules/loss.py (hunk 1 above)
Comment (MLgdg) [reply torch/nn/modules/loss.py]: its my  honor to get this suggestions from your

File: torch/nn/modules/loss.py (hunk 2 above)
Comment (MLgdg) [reply torch/nn/modules/loss.py]: its my honor to get this suggestions from you
